### ISSUE TITLE
Low mIoU but high accuracy 

### ISSUE THREAD

**Issue by minwang-ai** [ethnhe/FFB6D#41]
Hi Yisheng,

I get ride of 6D pose estimation related code and add mIoU as a second metric and remove the module of center, however, the mIoU is low while I accuracy is high.
Did you try to measure mIoU for semantic segmentation part?

Best,
Min

loss_rgbd_seg 0.10499326139688492
acc_rgbd 94.45703029632568
miou_rgbd 0.38043373823165894
val_loss 0.04918913063017036
loss_rgbd_seg 0.030238261446356773
acc_rgbd 96.937495470047
miou_rgbd 0.3542599678039551
val_loss 0.04686191983324519
loss_rgbd_seg 0.059770505875349045
acc_rgbd 95.51562070846558
miou_rgbd 0.3943084180355072
val_loss 0.04631547189674682
loss_rgbd_seg 0.12228231132030487
acc_rgbd 93.90234351158142
miou_rgbd 0.3640567660331726
val_loss 0.045014451784941296
loss_rgbd_seg 0.030247893184423447
acc_rgbd 97.1484363079071
miou_rgbd 0.3624090254306793
val_loss 0.0458199790344549
loss_rgbd_seg 0.1560874879360199
acc_rgbd 93.84374618530273
miou_rgbd 0.3667640686035156
val_loss 0.049276628012015176
loss_rgbd_seg 0.028056181967258453
acc_rgbd 97.04296588897705
miou_rgbd 0.3720839321613312
val_loss 0.0503167046118777

**Comment by minwang-ai** [comment]
Sorry! something was wrong before.
I reorganized the codes via pytorch lighting and new it is fine.
